boards/arm/colibri_imx7d_m4/doc/colibri_imx7d_m4.rst

```diff
@@ -0,0 +1,339 @@
+.. _colibri_imx7d_m4:
+
+NXP i.MX 7 Computer on Module - Colibri iMX7
+############################################
+
+Overview
+********
+
+The i.MX7 SoC is a Hybrid multi-core processor composed by Single/Dual Cortex A7
+core and Single Cortex M4 core.
+The Zephry was ported to run on the M4 core and the final goal is to have
```

Comment:
Change to:

    Zephyr was ported to run on the M4 core.  In a later release, it will also communicate
    with the A7 core (running Linux) via RPmsg.